nsxt/resource_nsxt_policy_predefined_gateway_policy.go

```diff
@@ -263,7 +264,22 @@ func createChildDomainWithGatewayPolicy(domain string, policyID string, policy m
 	return dataValue.(*data.StructValue), nil
 }
 
-func gatewayPolicyInfraPatch(context utl.SessionContext, policy model.GatewayPolicy, domain string, m interface{}) error {
+func gatewayPolicyInfraPatch(context utl.SessionContext, policy model.GatewayPolicy, domain string, m interface{}, withDomain bool) error {
```

Comment:
How are you using withDomain in this function?

Reply:
It's redundant, I'll remove that argument.